src/databricks/labs/ucx/source_code/known.py

```diff
@@ -173,6 +171,15 @@ def _analyze_dist_info(cls, dist_info_folder, known_distributions, library_root)
 
     @classmethod
     def _analyze_file(cls, known_distributions, library_root, dist_info, module_path) -> None:
+        # Avoiding circular import when rebuilding KnownList as FileLinter and its dependencies expect KnownList to
```

Comment:
Have a look at this. I consider this to be an exception as it is not part of ucx code path but a way for us to update the `known.json` in the source code